docs/animations/Light.md

```diff
@@ -23,7 +23,8 @@ The Light animation behavior performs a point light (A point source of light tha
     <behaviors:Light x:Name="LightBehavior" 
            Distance="10" 
            Duration="500" 
-           Delay="0" 
+           Delay="0"
+           EasingType="Bounce"
```

Comment:
Why bounce on all the docs? Why not empty or linear as was the previous

Reply:
It's for discoverability. The default has not changed, it's just the samples. We don't usually set the properties on the samples to the default, any reason why it should be in this case?

Reply:
Bounce is just such a weird animation that making it the default sample could confuse people about what will happen

Reply:
I could see that, changed it to Linear